design/sketch.webidl

```diff
@@ -25,6 +25,11 @@ dictionary GPUExtent3D {
 // ERROR HANDLING
 // ****************************************************************************
 
+interface GPUDeviceLostInfo {
+    readonly attribute DOMString reason;
+};
+callback GPUDeviceLostCallback = void (GPUDeviceLostInfo info);
```

Comment:
Can we add `lost` to the sketch.webidl file?

Reply:
Oops, forgot to update sketch.